### PR TITLE
Feature/csl837 fake messages logging

### PR DESCRIPTION
**Pull request by volhovM**
Mostly trivial changes related to:
* Add support for `log-warper-1.1.1`
* Hide some messages to secret logger

### CODE REVIEW

File: src/launcher/Main.hs [hunk 1]
```diff
@@ -5,30 +5,34 @@
...
-                                           metavar, option, progDesc, short, strOption)
-import           System.Directory         (getTemporaryDirectory)
-import qualified System.IO                as IO
-import           System.Process           (ProcessHandle)
-import qualified System.Process           as Process
-import           System.Timeout           (timeout)
-import           Turtle                   hiding (option, toText)
-import           Universum                hiding (FilePath)
+import           Control.Concurrent        (modifyMVar_)
+import           Control.Concurrent.Async  hiding (wait)
```
Comment (gromakovsky) [inline src/launcher/Main.hs]: Can you fix it while you are here?

File: src/launcher/Main.hs [hunk 2]
```diff
@@ -5,30 +5,34 @@
...
+                                            long, metavar, option, progDesc, short,
+                                            strOption)
+import           System.Directory          (getTemporaryDirectory)
+import           System.FilePath           ((</>))
+import qualified System.IO                 as IO
+import           System.Process            (ProcessHandle)
+import qualified System.Process            as Process
+import           System.Timeout            (timeout)
+import           System.Wlog               (lcFilePrefix)
+import           Turtle                    hiding (option, toText, (</>))
```
Comment (gromakovsky) [inline src/launcher/Main.hs]: And this one doesn't comply with guide too.

File: src/Pos/Block/Worker.hs [hunk 3]
```diff
@@ -72,7 +73,7 @@ blkOnNewSlotImpl :: WorkMode ssc m =>
 blkOnNewSlotImpl (slotId@SlotId {..}) sendActions = do
     -- Just ignore this line. It will be deleted after fake messages
     -- policy (CSL-837) is introduced.
-    logDebug "CSL-700 this message is top secret"
+    logDebugS "CSL-700 this message is top secret"
```
Comment (gromakovsky) [inline src/Pos/Block/Worker.hs]: Let's remove it maybe?

**Comment by gromakovsky** [comment]
It's not all sensitive data and it doesn't include fake logging. Is it enough for this issue?

**Comment by volhovM** [comment]
@gromakovsky yes, we've agreed that we won't introduce any fake logging for now because it's too complicated.

**Pull request merged**